### PR TITLE
feat(kibbeh): added settingsitembutton component

### PR DESCRIPTION
**Pull request by BronzW**

### CODE REVIEW

File: kibbeh/src/ui/SettingsItemButton.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <BaseSettingsItem className="p-4">
+      <p className="text-primary-200 mb-4">{text}</p>
+      <Button color="secondary-accent" size="small" onClick={onClick} disabled={disabled} className="py-2" >
+        <p className="font-bold">
+          {buttonText}
+        </p>
+      </Button>
+    </BaseSettingsItem>
+
```
Comment (dzmitry-vasileuski) [inline kibbeh/src/ui/SettingsItemButton.tsx]: Extra blank line

Comment (BronzW) [reply kibbeh/src/ui/SettingsItemButton.tsx]: Thank you, resolved in https://github.com/benawad/dogehouse/pull/2272/commits/524a88d1881d89c95920a4c92f92df36585fca94

**Comment by ajmnz** [comment]
Can you pull latest? We've just merged the secondary bg for Button

File: kibbeh/src/ui/SettingsItemButton.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        {text}
+      </p>
+      <Button
+        color="secondary-accent"
+        size="small"
+        onClick={onClick}
+        disabled={disabled}
+        className="py-2"
+      >
+        <p className="font-bold">
```
Comment (ajmnz) [inline kibbeh/src/ui/SettingsItemButton.tsx]: Remove the `p` tag and leave only the text as children

File: kibbeh/src/ui/SettingsItemButton.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+export const SettingsItemButton: React.FC<SettingsItemButtonProps> = ({
+  text,
+  buttonText,
+  disabled = false,
+  className = "",
+  onClick,
+}) => {
+  return (
+    <BaseSettingsItem className={`p-4 ${className}`}>
+      <p className="text-primary-200 mb-4">
```
Comment (ajmnz) [inline kibbeh/src/ui/SettingsItemButton.tsx]:
```suggestion
      <p className="text-primary-100 mb-4">
```

**Review by ajmnz — CHANGES_REQUESTED** [review]

File: kibbeh/src/ui/SettingsItemButton.tsx (hunk 3 above)
Comment (BronzW) [reply kibbeh/src/ui/SettingsItemButton.tsx:22]: > Can you pull latest? We've just merged the secondary bg for Button

Should I keep my change or was my change added already?

**Comment by BronzW** [comment]
> Can you pull latest? We've just merged the secondary bg for Button

Should I keep my change or was my change added already?

**Pull request merged**